portfolio.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Joseph Farmer</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+    <script src="main.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <header class="header">
+        <h1 class="header__title">Joseph Farmer</h1>
+        <nav class="navigate">
+            <ul class="header__menu">
+                <li class="header__menu-item">
+                    <a href="index.html">home</a>
+                </li>
+                <li class="header__menu-item">
+                    <a href="portfolio.html">Projects</a>
+                </li>
+                <li class="header__menu-item">
+                    <a href="resume.html">Resume</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <main class="main">
+        <h2 class="projects__header"> Projects</h2>
+        <div class="projects">
+            <h3 class="projects__header__1"> Garden Supply Store</h3>
+            <img class="projects__header__img-1" src="project-1.png" alt="project 1">
+            <article class="about-project-1">
+                <p>This is the first project that I completed on day one at Momemtum. It is a website composed of three pages
+                    for a made up garden supply store.</p>
+            </article>
+            <ul class="project-1-links">
+                <li class="project-1-link">
+                    <a href="https://hungry-harbor.glitch.me/" class="project-1-live">Live Site</a>
+                </li>
+                <li class="project-1-link">
+                    <a href="https://github.com/momentum-cohort-2018-07/lab-w1d3-Josephrfarmer/tree/master/homework-1" class="project-1-github">Code on GitHub</a>
+                </li>
+            </ul>
+            <div class="project_2">
```

Comment:
I notice there's a div around this project, but not the previous project. I would put one around both.